packages/forms/src/elements/MultiThumbRange.spec.tsx

```diff
@@ -438,7 +456,7 @@ describe('MultiThumbRange', () => {
       const { getAllByTestId } = render(<MultiThumbRange minValue={50} maxValue={40} />);
       const thumb = getAllByTestId('thumb')[1];
 
-      expect(thumb).toHaveStyle('left: 50px');
+      expect(thumb).toHaveStyle('left: 40px');
```

Comment:
Yes, in the new hook we picked a consistent approach for applying out-of-bounds logic – that is, start-to-end (the minimum will determine where the maximum can start rather than the other way around). The previous logic was inconsistent in `MultiThumbRange`.